src/types/generationContext/GenerationContext.ts

```diff
@@ -60,5 +61,6 @@ export type GenerationContext = {
   brand?: Brand;
   product?: Product;
   persona?: Persona;
+  imageUrl?: string;
```

Comment:
assuming we don't need validation for url in sdk, it should happen on the host app before passing in.
Just verifying.

Reply:
this url may change overtime (for now from polaris, maybe in future not from polaris) so I don't want to do any validation